src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveLoadNonFaultingUnOpTest.template

```diff
@@ -0,0 +1,242 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/******************************************************************************
+ * This file is auto-generated from a template file by the GenerateTests.csx  *
+ * script in tests\src\JIT\HardwareIntrinsics\Arm\Shared. In order to make    *
+ * changes, please update the corresponding template and run according to the *
+ * directions listed in the file.                                             *
+ ******************************************************************************/
+
+using System;
+using System.Numerics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void {TestName}()
+        {
+            var test = new LoadUnaryOpTest__{TestName}();
+
+            if (test.IsSupported)
+            {
+                // Validates basic functionality works
+                test.RunBasicScenario_Load();
+                test.RunBasicScenario_LoadNonFaulting();
+
+                // Validates calling via reflection works
+                // TODO-SVE: Enable once register allocation exists for predicates.
+                // test.RunReflectionScenario_Load();
```

Comment:
can we have a scenario for `ConditionalSelect(mask, LoadVectorByteNonFaultingZeroExtendToInt64(address), mergeValue)`? @a74nh ?

Reply:
I'm not sure if that statement is still true. It may have been fixed by the better handling we now have of predicates. Does this test work if you enable it?
If so, could you also enable the same test in `SveLoadMaskedUnOpTest.template` too please.